src/pages/About/Staff.js

```diff
@@ -0,0 +1,107 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import { useStaticQuery, graphql } from "gatsby";
+import { Fragment } from "react";
+
+import DividerLine from "../../components/DividerLine";
+import { colors, noHover } from "../../_Theme/UpdatedBrandTheme";
+import GridTrio from "../../components/GridTrio";
+
+const Staff = () => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const { contentfulTeam } = useStaticQuery(
+    graphql`
+      query {
+        contentfulTeam(contentful_id: { eq: "hjY1bbnBNQBvlSD21Me3K" }) {
+          name
+          members {
+            name
+            email
+            title
+            photo {
+              fluid(maxWidth: 322) {
+                srcSet
+                sizes
+              }
+              title
+              description
+            }
+          }
+        }
+      }
+    `
+  );
+  return (
+    <Fragment>
+      <GridTrio title={contentfulTeam.name}>
```

Comment:
`GridList` rather than `GridTrio` would be the component to use here. `GridList` will show a horizontal list for longer whereas `GridTrio` drops straight into one column when it gets squeezed